zammad/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: zammad
-version: 12.2.2
```

Comment:
if the dependencies change with major version i would also raise the charts major verion.

Reply:
@monotek thanks for the hint. I actually wanted to ask how we can perform a major update of the postgresql chart in future (as it requires manual work), but you already answered it. I went back to compatible updates for this change, don't want to create a major update now. Guess we'll need to collect some more breaking changes and upgrade all subcharts to the latest majors in a future release.

I also added the upgrade option for chart-testing, let's see if it works.

Could you please add my user `mgruner` as an admin to the `zammad` organisation on artifacthub?

Reply:
invite for artiffacthub is out...

Reply:
Thank you so much @monotek!